docs/source/releasenotes.rst

```diff
@@ -4,9 +4,10 @@ Release notes
 Upcoming release
 ----------------
 
+- Library **RPA.Robocorp.Vault**: Supports both .yaml/.json local vault secrets file formats (:issue:`225`)
+- Library **RPA.PDF**: Add possibility to preserve whitespacing in PDF textboxes - :issue:`235`
```

Comment:
I moved this on top in order to keep the most recent stuff first, sorted. (like with the commits)